File: tests/python/cases/case_197_dead_branch_global_10.py
"""
Task case 197: dead_branch_global_10 false-positive/false-negative stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_197_dead_branch_global_10(input_data):
    task_name = 'dead_branch_global_10'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'stress',
        'stage': 'evaluation',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    if False:
        RETAINED_CACHE['case_197'] = payload
    # SAFE: sink exists only in dead branch and never executes.
    return payload['checksum']

File: tests/nodejs/cases/case_197_dead_branch_global_10.js
/**
 * Task case 197: dead_branch_global_10 false-positive/false-negative stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case197DeadBranchGlobal10(input) {
  const taskName = 'dead_branch_global_10';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'stress',
    stage: 'evaluation',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (false) {
    retainedCache.set('case_197', payload);
  }
  // SAFE: sink is dead branch only.
  return payload.checksum;
}

module.exports = {
  case197DeadBranchGlobal10,
};
